README.md

```diff
@@ -90,8 +90,7 @@ After you're finished please remove all the comments and instructions!
 
 In order to run this project you need:
 
-- Node js installed on your computer
-- Npm Node package manager
+- Ruby installed on your computer
 - Terminal
 
```

Comment:
- [ ] As requested by the previous reviewer "_Since you have not deployed the project yet, Kindly remove the live demo link on your readme file to keep it professional as it is broken"_ your live demo link is still broken and giving a 404 error, having a professional documentation is also very important for our projects, please fix it :+1: 
- [ ] As mentioned above and also by the previous reviewer, you also need to fix the getting started section specifically the install one, as it is not a javascript project `npm install` will not do anything replace it with `bundle install`
![image](https://user-images.githubusercontent.com/97521557/219094122-8d6eb9b6-399d-4ab4-99e4-930369acf22f.png)

- [ ] Also `npm start ` will throw an error, a user can run this program locally with `ruby rubyfile.rb`
![image](https://user-images.githubusercontent.com/97521557/219094888-205a9e3c-203d-46de-9e0f-2b8af3f77707.png)

- [ ] Same thing for test and deploy you run test with rspec and as it is a console app, there is no deploy\

- [ ] Also mentioned by the previous reviewer "_Kindly adjust the link to your MIT license as it is broken to keep your repo professional" your Mit link is still broken and throwing a 404, add a LICENSE file to the root of your project and fix it 

I hope things are much clear now, please implement the fixes :pray: